mod/metrics/set/ltype_rtype/metric/wikirate_company/record.rb

```diff
@@ -0,0 +1,118 @@
+format :html do
+  def all_answers
+    @result ||=
+      Answer.fetch({ record_id: card.id }, sort_by: :year, sort_order: :desc)
+  end
```

Comment:
I wasn't sure if `@result` gets cleared if it is a class instance variable on `Answer`